config-connector/solutions/projects/kpt/simple-project/README.md

```diff
@@ -0,0 +1,49 @@
+Simple Project
+==================================================
+
+# NAME
+
+  simple-project
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create a project in an organization.
+
+# CONSUMPTION
+  Using kpt:
+  ```
+  BASE=https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+  kpt pkg get $BASE/config-connector/solutions/projects/kpt/simple-project simple-project
+  ```
+
+# REQUIREMENTS
+  A working cluster with Config Connector installed.
+  
+  Cloud Resource Manager API and Cloud Billing API enabled in the project where Config Connector is installed.
+  
+  The cnrm-system service account must have `roles/resourcemanager.projectCreator` in your organization and `roles/billing.user` for your billing account.
+  
+# USAGE
+  In order to use, replace the `${BILLING_ACCOUNT_ID?}` and `${ORG_ID?}` values with your billing account and organization id. You can do this with kpt setters or by manually editing the YAML files. To use kpt setters:
+  ```
```

Comment:
"You can do this with kpt setters or by manually editing the YAML files. To use kpt setters:" -> "You can do this with kpt setters:"

Let's not suggest manual edits if it is not required.